src/model/element-base.ts

```diff
@@ -61,15 +61,16 @@ export abstract class ScannedElementBase implements Resolvable {
       }
       this.jsdoc =
           this.jsdoc ? jsdoc.join(commentJsdoc, this.jsdoc) : commentJsdoc;
+      this.applyJsdocDemoTags();
```

Comment:
All the other classes extend ElementBase, can we remove all the other calls to applyJsdocDemoTags()?